CHANGELOG.md

```diff
@@ -9,9 +9,10 @@
   - [#86](https://github.com/Datatamer/unify-client-python/issues/86) Add support for creating projects
   - [#114](https://github.com/Datatamer/unify-client-python/issues/114) Add support for generating pairs estimate
   - [#106](https://github.com/Datatamer/unify-client-python/issues/106) Add support for initializing a source dataset
-  
+
   **BUG FIXES**
   - [#118](https://github.com/Datatamer/unify-client-python/issues/118) Fix JSON sent for Dataset.update_records
+  - [#123] (https://github.com/Datatamer/unify-client-python/projects/1) Fix base_path bug for a custom api endpoint
```

Comment:
This needs to point to the issue, not the project.

Reply:
done